pkg/util/ipvs/ipvs.go

```diff
@@ -123,28 +120,15 @@ func (rs *RealServer) Equal(other *RealServer) bool {
 		rs.Port == other.Port
 }
 
-// GetKernelVersionAndIPVSMods returns the linux kernel version and the required ipvs modules
-func GetKernelVersionAndIPVSMods(Executor exec.Interface) (kernelVersion string, ipvsModules []string, err error) {
-	kernelVersionFile := "/proc/sys/kernel/osrelease"
-	out, err := Executor.Command("cut", "-f1", "-d", " ", kernelVersionFile).CombinedOutput()
-	if err != nil {
-		return "", nil, fmt.Errorf("error getting os release kernel version: %v(%s)", err, out)
-	}
-	kernelVersion = strings.TrimSpace(string(out))
-	// parse kernel version
-	ver1, err := version.ParseGeneric(kernelVersion)
-	if err != nil {
-		return kernelVersion, nil, fmt.Errorf("error parsing kernel version: %v(%s)", err, kernelVersion)
-	}
+// GetRequiredIPVSModules returns the required ipvs modules for the given linux kernel version.
+func GetRequiredIPVSModules(kernelVersion *version.Version) []string {
 	// "nf_conntrack_ipv4" has been removed since v4.19
 	// see https://github.com/torvalds/linux/commit/a0ae2562c6c4b2721d9fddba63b7286c13517d9f
-	ver2, _ := version.ParseGeneric("4.19")
+	criticalVers := version.MustParseGeneric("4.19")
```

Comment:
Why name `criticalVers`?

Reply:
it is changing behavior so a critical point but i can change it to `ver`?

Reply:
I think something more descriptive would be nice. Maybe something like `versionNfConntrackIPv4Removed`?

Reply:
the comment is up one line and the scope is very limited so if it is ok for you, i think just `ver` is enough?

Reply:
How about this?
```go
if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
		ipvsModules = append(ipvsModules, ModIPVS, ModIPVSRR, ModIPVSWRR, ModIPVSSH, ModNfConntrackIPV4)
		return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrackIPV4}
	} 

	return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrack}
```

Reply:
Yes that is even better =)